generate_schedule.py

```diff
@@ -13,25 +13,20 @@
 EPS = 1e-8
 
 
-def generate_flex_band(scenario, core_standing_time=None):
+def generate_flex_band(scenario, gc_index, core_standing_time=None):
```

Comment:
Use grid connector ID instead of index. This is easier, better to read and cleaner, as GC are saved in a dict and not in a list).